Assets/Plugins/PlayroomPlugin.jslib

```diff
@@ -12,6 +12,7 @@ mergeInto(LibraryManager.library, {
     // onLaunchCallback
   ) {
     function embedScript(src) {
+      script.crossOrigin = 'anonymous';
```

Comment:
tested this? does this cause any issues?

Reply:
Requested and tested by a user here #25. 
Does not break the SDK in any way, however, the security of it is debatable. We shouldn't allow **all** origins I suppose.
Will keep this under watch.